modelon/impact/client/entities/workspace.py

```diff
@@ -26,6 +28,42 @@
 ]
 
 
+class WorkspaceDefinition:
+    def __init__(self, data: Dict[str, Any]):
+        self.data = data
+
+    def to_dict(self) -> Dict[str, Any]:
+        return self.data
+
+    def to_file(self, path: Optional[str] = None) -> str:
```

Comment:
I think we can skip having path optional here to simplify. Not sure the user wants to write the definition to file in an unknown location.